front/css/dark-patch.css

```
@@ -14,11 +14,20 @@
   --color-aqua: #00c0ef;
   --color-lightblue: #3c8dbc;
   --color-blue: #0060df;
+  --color-black: #000;
   --color-white: #fff;
   --color-green: #00a65a;
   --color-yellow: #f39c12;
   --color-red: #dd4b39;
   --color-gray: #8c8c8c;
+  --color-lightgray: #bec5cb;
+  --color-white: #fff;
```

Stylelint (17.13.0)

[error] Duplicate custom property "--color-white" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)